src/maps/cc-logsmap.js

```diff
@@ -83,15 +86,84 @@ export class CcLogsMap extends LitElement {
     this.loading = false;
     this.mode = 'points';
     this.viewZoom = 2;
+    this._points = [];
+    this._pointsByCoords = {};
   }
 
   /**
-   * Add several points to the map for the live blinking dots mode.
+   * Add several points to the map with blinking dots markers.
    * @param {Point[]} points - List of points.
    * @param {PointsOptions} options - Options to spread the display of the different points over time.
    */
-  addPoints (...params) {
-    this.shadowRoot.querySelector('cc-map').addPoints(...params);
+  addPoints (points, options = {}) {
+
+    const { spreadDuration = false } = options;
+
+    const timeStep = (spreadDuration !== false)
+      ? Math.floor(spreadDuration / points.length)
+      : 0;
+
+    points.forEach((p, i) => {
+      setTimeout(() => this._addPoint(p), timeStep * i);
+    });
+  }
+
+  _addPoint ({ lat, lon, count = 1, delay = 1000, tooltip }) {
+
+    const coords = [lat, lon].join(',');
+    const newPoint = { lat, lon, count, tooltip };
+
+    // Add point now
+    if (this._pointsByCoords[coords] == null) {
+      this._pointsByCoords[coords] = [];
+    }
+    this._pointsByCoords[coords].push(newPoint);
+
+    this._updatePoints();
+
+    // Schedule delete point
+    setTimeout(() => {
```

Comment:
Don't you want to abort the timeout when the component is "dropped"?

Reply:
You mean if the component is removed from the DOM?
It's the code we had in `<cc-map>`, as is, I just move it.
Maybe we could investigate this later...

https://github.com/CleverCloud/clever-components/issues/158

Reply:
> You mean if the component is removed from the DOM?

Yes. I don't like having timeouts that won't do anything anymore running in the background. Sure, let's check it later :+1: